front-end/src/common/card-editor/CardEditor.jsx

```diff
@@ -16,9 +16,12 @@ const sectionIds = [0, 1, 2];
 
 const HeartIcon = () => <img src={heart} width="25px" height="25px" />;
 
-function CardEditor({ form = {}, setForm, templateData }) {
+function CardEditor({ form = {}, setForm, templateData, shouldDisableTour }) {
```

Comment:
```suggestion
function CardEditor({ form = {}, setForm, templateData, shouldDisableTour = false }) {
```
if you set up the function signature like this, you don't need the if statement in line 22!